__tests__/app.test.js

```diff
@@ -190,4 +190,47 @@ describe("GET /api/articles/:article_id/comments",() => {
             expect(body.message).toBe('Path not found')
         })
     })
+})
+
+describe("POST /api/articles/:article_id/comments",()=>{
+    test("POST: 201 inserts a new comment to the db and responds with the posted comment back to the user",() => {
+        const newComment = {
+            "username" : "butter_bridge",
+            "body"  : "The article have given me some beautiful insights into the world of ecommerce"
+        }
+        return request(app)
+        .post('/api/articles/2/comments')
+        .send(newComment)
+        .expect(201)
+        .then(({body})=>{
+            expect(body.comment.comment_id).toBe(19)
+            expect(body.comment.author).toBe('butter_bridge')
+            expect(body.comment.body).toBe("The article have given me some beautiful insights into the world of ecommerce")
+        })
+    })
+    test("POST: 400 responds with an appropriate status code and error message when provided with an invalid comment(no author)" ,() => {
+        const newComment = {
+            "body" : "The world is round around me" 
+        }
+        return request(app)
+        .post('/api/articles/1/comments')
+        .send(newComment)
+        .expect(400)
+        .then(({body}) => {
+            expect(body.message).toBe('Bad request')
+        })
+    })
+    test("POST: 400 responds with the appropriate status code and error message when provided valid fields but invalid values to the fields ",() => {
+        const newComment = {
+            "username" : 23,
+            "body" : 123
+        }
+        return request(app)
+        .post('/api/articles/99/comments')
+        .send(newComment)
+        .expect(400)
+        .then(({body}) => {
+            expect(body.message).toBe("Bad request")
+        })
+    })
```

Comment:
A few things to consider for testing here:

-What if they try to use a username that doesn't exist in the database?
-What if there are extra keys on the newComment object?